diff --git a/pyproject.toml b/pyproject.toml
index ef75703..8e9fa79 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -44,6 +44,10 @@ dependencies = [
 # must be able to be installed simultaneously.
 runner = [
   "snakemake>=9.3.3",
+  # open-meteo API dependencies
+  "openmeteo-requests",
+  "requests-cache",
+  "retry-requests",
   # Add additional dependencies for new modules and ETL process here.
 ]
 
@@ -242,6 +246,8 @@ mkdocs-inline-select-svg-plugin = { pip = true }
 mkdocs-mermaid2-plugin = { pip = true }
 mkdocs-section-index = { pip = true }
 mkdocstrings = { skip = true, packages = ["mkdocstrings-python>=1.16.10"] }
+openmeteo-requests = { pip = true }
+retry-requests = { pip = true }
 snakemake = { channel = "bioconda"}
 snakefmt = { channel = "bioconda"}
 
